connect-examples/v2/php_checkout/templates/example-curl-helpers.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/*******************************************************************************
+ * Create constants for the Square Endpoint URLs
+ ******************************************************************************/
+
+// Location - used to query the location ID for the store
+if (!defined(_SQ_ENDPOINT_LOCATION)) {
+  define(
+    "_SQ_ENDPOINT_LOCATION",
+    "https://connect.squareupstaging.com/v2/locations"
```

Comment:
Not sure if I'm missing context, but should these links point to connect.squareup.com instead of connect.squareupstaging.com?